lib/modules/platform/bitbucket/utils.ts

```diff
@@ -1,56 +1,27 @@
 import url from 'url';
+import { defaultSchema } from '@atlaskit/adf-schema/schema-default';
+import {
+  JSONDocNode,
+  JSONTransformer,
+} from '@atlaskit/editor-json-transformer';
```

Comment:
please don't add new deps for functionality which is already provided by other existing deps.

checkout remark

Reply:
I had already looked through the [available remark plugins](https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins) but there isn't any that support converting into [Atlassian Document Format](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/), which is an Atlassian extension of [prosemirror](https://prosemirror.net/) for rich text fields.  

See [JRACLOUD-77436](https://jira.atlassian.com/browse/JRACLOUD-77436) for current "best" approach to convert markdown -> ADF as recommended by Atlassian

Reply:
That said, I'll play around with downgrading to the v2 Jira Cloud API to see if I can get sufficient rich text formatting for the description/body via markdown 🤞

Reply:
So I went down the Jira v2 API route, and have it functionally working without any additional packages/deps, but the rendering of issue body was pretty limited as it used the much dated [Atlassian Text Formating Notation](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all).

I then returned back to the Jira v3 API and am using remark to get the Github Markdown AST and have implemented a custom AST parser to generate the Atlassian Document Format needed.  More custom code, but at least gets us away from needing the atlaskit packages to do this automagically.